hack/slurm/README.md

```diff
@@ -0,0 +1,95 @@
+# Slurm deployment with Slink
+
+[Slink](https://slinky.ai) is a project from SchedMD, and one of its open source tools
+is [slurm-operator](https://github.com/SlinkyProject/slurm-operator).
+
+Please [this presentation](https://slurm.schedmd.com/SLUG24/Slinky-Slurm-Operator.pdf) for its overview and future plan.
+
+To deploy Slurm Operator and create a Slurm cluster,
+follow [the quick start guide](https://github.com/SlinkyProject/slurm-operator/blob/main/docs/user/quickstart.md)
+
+The above steps will create the following `Cluster` resource and `NodeSet` resource.
+
+```yaml
+kind: Cluster
+metadata:
+  name: slurm
+  namespace: slurm
+spec:
+  server: http://slurm-restapi.slurm:6820
+  token:
+    secretRef: slurm-token-slurm
+```
+
+```yaml
+apiVersion: slinky.slurm.net/v1alpha1
+kind: NodeSet
+metadata:
+  name: slurm-compute-debug
+  namespace: slurm
+spec:
+  clusterName: slurm
+  replicas: 1
+  serviceName: slurm-compute
+  minReadySeconds: 0
+  updateStrategy:
+    rollingUpdate:
+      maxUnavailable: 20%
+      partition: 0
+      paused: false
+    type: RollingUpdate
+  persistentVolumeClaimRetentionPolicy:
+    whenDeleted: Retain
+  selector:
+    matchLabels:
+      app.kubernetes.io/instance: slurm-compute-debug
+      app.kubernetes.io/name: slurmd
+  template:
+    ...
+    spec:
+      hostname: slurm-compute-debug
+      priorityClassName:
```

Comment:
This field is empty; is that alright?

Reply:
That's correct. This might be because I rendered this with `helm template --debug`.